bots/tests-scan

```diff
@@ -49,7 +49,7 @@ DEFAULT_VERIFY = {
 # Non-public images used for testing
 REDHAT_VERIFY = {
     "verify/rhel-7": [ "master", "pulls" ],
-    "verify/rhel-7-4": [ ],
+    "verify/rhel-7-4": [ "pulls" ],
```

Comment:
This needs to say "master", "rhel-7.4", or similar. Ditto for adjoining lines. Should we do this as a follow up?